Formula/circleci.rb

```diff
@@ -32,11 +32,8 @@ def install
     ]
     system "go", "build", *std_go_args(ldflags: ldflags)
 
-    output = Utils.safe_popen_read("#{bin}/circleci", "--skip-update-check", "completion", "bash")
-    (bash_completion/"circleck").write output
-
-    output = Utils.safe_popen_read("#{bin}/circleci", "--skip-update-check", "completion", "zsh")
-    (zsh_completion/"_circleci").write output
+    generate_completions_from_executable(bin/"circleci", "--skip-update-check", "completion",
+                                        shells: [:bash, :zsh])
```

Comment:
Currently, the `basename` isn't the same for both shells. Bash uses `circleck` and zsh uses `circleci`

Reply:
It works, nevertheless. Also, I suppose the `circleck` in the bash base name has been a typo when adding the completions previously.